python/pyiceberg/catalog/rest.py

```diff
@@ -490,6 +501,36 @@ def rename_table(self, from_identifier: Union[str, Identifier], to_identifier: U
 
         return self.load_table(to_identifier)
 
+    def commit_table(
+        self, identifier: Union[str, Identifier], updates: Tuple[TableUpdate, ...], requirements: Tuple[TableRequirement, ...]
+    ) -> TableResponse:
+        """Updates the table
+
+        Args:
+            identifier (str | Identifier): Namespace identifier
+            updates (Tuple[TableUpdate]): Updates to be applied to the table
+        Raises:
+            NoSuchTableError: If a table with the given identifier does not exist
+        """
+        payload = CommitTableRequest(updates=updates).json()
```

Comment:
Missing requirements here.

Reply:
I just noticed this as well. Added them here and in all the `commit_table` instances.